site/en/docs/privacy-sandbox/fledge-best-practices/index.md

```diff
@@ -0,0 +1,100 @@
+* Combining similar interest groups will decrease the number of times `generateBid()` must be run. An interest group's `userBiddingSignals` property can be used to store additional metadata about the user, so fewer interest groups doesn't have to mean less effective targeting.
+* Currently, FLEDGE supports seller-specified limits on the numbers of interest groups,and an API for buyers to specify the relative priority of their interest groups. This can be used to significantly reduce the number of bidding scripts to run.
+
+There is ongoing discussion about how to filter the interest groups that participate in an auction. If you have ideas in this area, [file a GitHub issue](https://github.com/WICG/turtledove/issues/new) or contribute to [Issue 305](https://github.com/WICG/turtledove/issues/305).
+
+There is ongoing discussion about how to reuse the JavaScript execution environment between auctions. Reusing it reduces the amount of time the browser must spend initializing them. You can follow along on [Issue 304](https://github.com/WICG/turtledove/issues/304) or [Issue 310](https://github.com/WICG/turtledove/issues/310), or [file a new GitHub issue](https://github.com/WICG/turtledove/issues/new).
+
+### Reuse bidding scripts
+
+Interest groups sharing a bidding script can prevent the browser from having to download multiple scripts (which incurs extra network requests) and parse and compile them.
+
+### Reuse `trustedBiddingSignalsUrls`
+
+Network latency and resource usage can be very significant. Fewer real-time trusted bidding signals fetches can help reduce this latency.
+
+As the explainer notes, the trusted bidding [signal fetches can be combined](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#11-joining-interest-groups) when the `trustedBiddingSignalsUrl` is reused amongst multiple interest groups, so be sure to use the same `trustedBiddingSignalsUrl` between interest groups when possible.
+
+### Smaller real-time trusted bidding signals fetches
+
+Network latency can be very significant, and this is directly impacted by how much data is transferred during the real-time trusted bidding signal fetches. Prefer storing ad-specific or interest-group-specific data in the interest group, rather than on the real-time trusted bidding signal server. Reserve the real-time trusted bidding signal data for only those truly real-time signals, like campaign budgeting or kill-switches.
+
+Any signal that can be updated on a daily or longer basis should be stored in the interest group and updated via the daily updates.
+
+## Seller best practices
+
+Make sure you're monitoring and optimizing for FLEDGE auction efficiency.
+
+### Monitor your auctions
+
+Collect metrics on your auctions:
+
+* As discussed above, the number of interest group owners and interest groups bidding (both visible metrics to `scoreAd()`) have a significant effect on auction performance and are worth monitoring.
```

Comment:
```suggestion
* The number of interest group owners and interest groups bidding significantly impacts auction performance and should be monitored. The metrics are visible in `scoreAd()`.
```

Reply:
Someone may not read the above if they're only interested in the seller side. That said, you could link above to the corresponding issue.